src/reducers/form-actions-reducer.js

```diff
@@ -230,7 +230,7 @@ export function createFormActionsReducer(options) {
         let errors = { ...fieldState.errors };
         let valid;
 
-        if (action.omitKeys) {
+        if (action.omitKeys && action.omitKeys.length > 0) {
```

Comment:
This was actually a bad idea. Styll would have same error when fieldState.errors is string, and omitKeys is nonempty array.

Better solution:
```javascript
      case actionTypes.RESET_VALIDITY: {
        let validity = { ...fieldState.validity };
        let errors;
        let valid;

        if (action.omitKeys && typeof fieldState.errors !== 'string') {
          errors = { ...fieldState.errors };
          action.omitKeys.forEach(key => {
            delete validity[key];
            delete errors[key];
          });
          valid = isValidityValid(validity);
        } else {
          validity = customInitialFieldState.validity;
          errors = customInitialFieldState.errors;
          valid = customInitialFieldState.valid;
        }

        fieldUpdates = {
          valid,
          validity,
          errors
        };

        subFieldUpdates = {
          valid: customInitialFieldState.valid,
          validity: customInitialFieldState.validity,
          errors: customInitialFieldState.errors
        };

        break;
      }
```